nomad/structs/structs.go

```diff
@@ -5610,7 +5614,10 @@ func (p *PeriodicConfig) Validate() error {
 	}
 
 	var mErr multierror.Error
-	if p.Spec == "" {
+	if p.Spec != "" && len(p.Specs) != 0 {
+		_ = multierror.Append(&mErr, fmt.Errorf("You can use only cron or crons"))
```

Comment:
```suggestion
		_ = multierror.Append(&mErr, fmt.Errorf("Only cron or crons may be used"))
```

Should've noticed this as well 😅 

We usually avoid error messages pointing to a specific subject, like `you`.

Reply:
Very good suggestion.